Stream_support/include/CGAL/IO/STL.h

```diff
@@ -222,6 +222,31 @@ bool read_STL(const std::string& fname,
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 // Write
 
+namespace internal {
+
+template <typename K>
+typename K::Vector_3 construct_normal_of_STL_face(const typename K::Point_3& p,
```

Comment:
What about calling it `CGAL::PMP::approximate_normal()`   or similar. Or does that open a can of worms?

Reply:
We do have `PMP::internal::normalize()`, which uses `approximate_sqrt()`, but that would make `Stream_support` depend on `PMP`.

Reply:
It would be really handy to have an `approximate_normal()` function globally, (or outside PMP)

Reply:
`Kernel_3::Compute_approximate_angle_3` has already introduced inexact kernel functors (due to the trigonometry), so this could be an idea.